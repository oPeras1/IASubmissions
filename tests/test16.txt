30	34	34	35	35	37	37	37	37	38	38	38	36	36	36
30	34	34	35	35	37	37	37	32	32	38	38	36	36	36
30	34	34	26	35	31	31	31	32	32	33	33	33	29	36
30	25	25	26	26	26	31	32	32	27	24	33	29	29	29
22	25	25	26	26	23	27	27	27	27	24	28	28	29	29
22	22	25	26	26	23	27	27	24	24	24	28	28	28	19
22	23	23	23	23	23	20	20	20	20	24	21	21	19	19
17	17	17	18	18	18	15	20	20	21	21	21	21	19	19
17	17	18	18	18	18	15	15	15	15	15	16	16	16	19
13	13	13	13	10	10	11	11	11	15	12	16	16	14	14
13	10	10	10	10	10	11	11	11	12	12	14	14	14	14
8	10	2	2	2	2	9	9	11	12	12	5	5	6	6
8	8	8	2	3	2	9	9	9	7	7	5	5	5	6
1	1	2	2	3	2	3	4	4	4	7	7	7	5	6
1	1	1	2	3	3	3	4	4	4	5	5	5	5	6