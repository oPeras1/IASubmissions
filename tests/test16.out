I	34	T	L	L	I	I	I	I	L	L	L	T	T	T
I	T	T	35	L	37	37	37	32	S	38	L	36	T	36
I	34	T	26	L	T	T	T	S	S	T	T	T	S	36
I	L	L	26	T	26	T	32	S	27	24	T	29	S	S
T	25	L	T	T	23	L	L	L	27	T	S	S	29	S
T	T	L	26	T	23	L	27	24	T	T	28	S	S	I
T	23	I	I	I	I	T	T	T	20	T	21	L	19	I
L	L	L	18	T	18	15	T	20	21	L	L	L	19	I
L	17	18	T	T	T	15	I	I	I	I	16	S	S	I
I	I	I	I	10	L	S	S	11	15	T	S	S	14	L
13	10	10	L	L	L	11	S	S	T	T	14	L	L	L
L	10	2	I	2	2	9	T	11	12	T	S	S	6	I
L	L	L	I	3	2	T	T	T	7	L	5	S	S	I
1	T	2	I	3	2	L	4	S	S	L	L	L	5	I
T	T	T	I	L	L	L	S	S	4	5	5	5	5	I